src/Business Foundation/App/NoSeries/src/Legacy/NoSeriesMgt.Codeunit.al

```diff
@@ -0,0 +1,541 @@
+﻿// remove
+
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+codeunit 281 NoSeriesMgt
```

Comment:
Will MS apply some common 'styling' rules to sort properties, sort variables etc?

Apply below styling code actions, still shows quit some files to be updated.

```
"alOutline.codeCleanupActions": [
        "AddMissingParentheses",
        "AddObjectCaptions",
        "AddTableFieldCaptions",
        "AddToolTips",
        "AddPageFieldCaptions",
        "ConvertObjectIdsToNames",
        "FixIdentifiersCase",
        "FixKeywordsCase",
        "FormatDocument",
        "LockRemovedFieldCaptions",
        "RemoveBeginEnd",
        "RemoveUnusedVariables",
        "RemoveWithStatements",
        "SortPermissions",
        "SortPermissionSetList",
        "SortProperties",
        "SortVariables",
        "TrimTrailingWhitespace",
        "RemoveEmptyLines",
        "RemoveEmptySections",
        "RemoveEmptyTriggers",
        "RemoveRedundantAppAreas" 
    ],
```
![image](https://github.com/microsoft/BCApps/assets/25268332/065b1545-0036-4a74-b408-e4d3ed5461d0)

Reply:
I have applied these except SortVariables, that one needs more discussion. The others make a lot of sense.